plot.py

```diff
@@ -562,12 +564,13 @@ def _update_plot(self):
                                 ax=self.ax, title=self.title)
             self.display.set_limits(xlim=(self.limits['xmin'], self.limits['xmax']),\
                                     ylim=(self.limits['ymin'], self.limits['ymax']),\
-                                    ax=self.ax)
+                                    ax=self.ax, fig=self.fig)
             self.display.plot_grid_lines()
         else:
             self.display = pyart.graph.RadarDisplay(self.Vradar.value)
             if self.Vradar.value.scan_type != 'rhi':
                 # Create Plot
+                #XXX this should be handel else where
```

Comment:
Yes, I do think this is the wrong place for this testing. Also Display should not be overwriting the Vtilt value, its responsibility of the function that is changing Vtilt to assure it has a valid value (0<=value<=len(tilts))